docs/quick_start/index.md

```diff
@@ -0,0 +1,118 @@
+
+Let's set up a local Kubernetes cluster, which is a vital part of our development environment. The `dev-cluster` directory contains configuration files for various services that will be deployed to our local Kubernetes cluster. These services include:
+
+- **PostgreSQL Database**: A relational database system for storing and managing application data.
+- **Blob Storage**: A service for storing large amounts of unstructured data, like images or videos.
+- **RabbitMQ**: A message broker that enables applications to communicate with each other asynchronously.
+- **Kratos**: An identity management system that handles user accounts, authentication, and authorization.
+- **Seq**: Log system designed for searching, analysis, and visualization of structured log data.
+- **Jaeger**: An open-source tool for tracing requests.
+- **Traefik**: A cloud-native application proxy that routes incoming requests to the appropriate microservices.
+- **OpenTelemetry**: A set of APIs, SDKs, and tools to generate, collect, and export telemetry data.
+- **Metabase**: Tool for visualizing and sharing data insights.
+
+To configure the local cluster, start by navigating to the `dev-cluster` directory. Once there, ensure that you've set the `sendgrid_api_key` in the `terraform.tfvars` file. Without this api key, the functionality for sending emails won't be available:
+
+```terraform
+sendgrid_api_key = "my_sendgrid_api_key"
+```
+
+Then, execute the following script to set up the local cluster:
+
+```sh
+./deploy.sh
+```
+
+## Setting up the application
+
+After successfully setting up the local cluster, your next step is to initiate your application. Begin by navigating to the `backend` directory. This directory is the heart of your application, containing various subdirectories and files, each serving a specific purpose:
+
+```txt
+backend
+├── dev - Configuration for local development.
+├── release - Dockerfiles for production deployment.
+├── src
+│   ├── Apps
+│   │   ├── ProjectName.Api - Hosts the API, serving as the backend entrypoint.
+│   │   ├── ProjectName.LeanPipeFunnel - Manages real-time notifications.
+│   │   └── ProjectName.Migrations - Handles database migrations.
+│   └── ContextName
+│       ├── ProjectName.ContextName.Contracts - API client interfaces.
+│       ├── ProjectName.ContextName.Domain - Core business logic.
+│       └── ProjectName.ContextName.Services - Service layer.
+├── tests
+│   ├── ContextName
+│   │   ├── ProjectName.ContextName.Domain.Tests - Tests for business logic.
+│   │   └── ProjectName.ContextName.Services.Tests - Tests for service layer.
+│   ├── ProjectName.IntegrationTests - Comprehensive application-wide tests.
+```
+
+### Generating initial migrations
+
+Let's start with generating initial migrations. Begin by navigating, to the `backend/src/Apps/ProjectName.Migrations` directory. Generate initial migrations using the `dotnet ef` tool (install it via `dotnet tool install --global dotnet-ef`):
```

Comment:
I talked with Krzysiek about this one, this tool IMO should be put in the `dotnet-tools.json`. /cc: @Saancreed

Reply:
I will update this when it will be in `dotnet-tools.json`